com.unity.netcode.gameobjects/Runtime/NetworkVariable/OwnerNetworkVariable.cs

```diff
@@ -0,0 +1,37 @@
+using System;
+
+namespace Unity.Netcode
+{
+    [Serializable]
+    public class OwnerNetworkVariable<T> : NetworkVariable<T> where T : unmanaged
```

Comment:
I like the feature, the simplicity of the implementation is great. How you went about it is super tactical. But...

Using inheritance for it is, IMO, wrong. Inheritance comes with a huge maintenance cost, changing the base or derive class can introduce unintended effects. I think it also breaks the SOLID open–closed principle 
> "software entities (classes, modules, functions, etc.) should be open for extension, but closed for modification"  
Here, I see a derived class implementing modification.

Could we have a boolean in the base class and the code changed to take either behaviour based on the member variable ? This would also have the benefit of allowing the transition of one type of NV to the other at runtime.